<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigSectionInterface</class>
 <widget class="QWidget" name="ConfigSectionInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="13" column="0" colspan="3">
    <widget class="QLabel" name="labelRestart">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>This modification will be applied during the next start of the software.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Decorations</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Unique instance</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Sort divisions</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QComboBox" name="comboColorTheme">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboLangue">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <layout class="QGridLayout" name="gridLayoutColors" columnstretch="0,2,2,2,1,1">
     <item row="1" column="0">
      <widget class="QLabel" name="labelColorText">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Text</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushColorListBackground">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border: 1px solid #888;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushColorWindowText">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border: 1px solid #888;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelColorWindow">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Window</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushColorWindowBackground">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border: 1px solid #888;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushColorButtonText">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border: 1px solid #888;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushColorButtonBackground">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border: 1px solid #888;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelColorButton">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Button</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelColorBackground">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Background</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushColorSelectionText">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border: 1px solid #888;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="labelColorSelection">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Selection</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushColorSelectionBackground">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border: 1px solid #888;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="pushColorListAlternativeBackground">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border: 1px solid #888;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QLabel" name="labelColorList">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>List and table</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4" colspan="2">
      <widget class="QPushButton" name="pushColorListText">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border: 1px solid #888;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="2">
    <widget class="QComboBox" name="comboStyle">
     <item>
      <property name="text">
       <string notr="true">Fusion</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="labelSubTitle1">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QLabel" name="labelSubTitle3">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Theme</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="labelColors">
     <property name="text">
      <string>Colors</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="labelSubTitle2">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Behaviour</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QCheckBox" name="checkDecorations">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Style</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_22">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Key names</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboKeyName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Middle C → 60, then 61</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Middle C → C3, then C♯3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Middle C → C3, then D♭3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Middle C → C4, then C♯4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Middle C → C4, then D♭4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Middle C → C5, then C♯5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Middle C → C5, then D♭5</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="comboSortDivisions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>by key range</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>by velocity range</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>in alphabetical order</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="speaking of the &quot;sort divisions&quot;">none</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="checkUniqueInstance">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="labelBorders">
     <property name="text">
      <string>Window borders</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QCheckBox" name="checkBorders">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboLangue</tabstop>
  <tabstop>comboKeyName</tabstop>
  <tabstop>comboSortDivisions</tabstop>
  <tabstop>checkUniqueInstance</tabstop>
  <tabstop>comboStyle</tabstop>
  <tabstop>comboColorTheme</tabstop>
  <tabstop>checkDecorations</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
